### ISSUE TITLE
[Bug]: I can't access my save on my phone

### ISSUE THREAD

**Issue by Bluebloodhero** [MelvorIdle/melvoridle.github.io#1684]
### Describe the bug

populateMasteryObject@https://ios.melvoridle.com/assets/js/game/mastery2.js?1290:4:31i
updateWindow@https://ios.melvoridle.com/assets/js/game/main.js?1290:4:264
finaliseLoad@https://ios.melvoridle.com/assets/js/game/main.js?1290:372:13
selectCharacter@https://ios.melvoridle.com/assets/js/game/main.js?1290:364:192
onloadEvent@https://ios.melvoridle.com/assets/js/game/save.js?1290:61:232
https://ios.melvoridle.com/assets/js/game/save.js?1290:66:382
https://cdn.melvor.net/core/v018/assets/js/oneui.core.min.js:6:84826
each@https://cdn.melvor.net/core/v018/assets/js/oneui.core.min.js:6:2971
https://cdn.melvor.net/core/v018/assets/js/oneui.core.min.js:6:84807
r@https://cdn.melvor.net/core/v018/assets/js/oneui.core.min.js:6:28088
fireWith@https://cdn.melvor.net/core/v018/assets/js/oneui.core.min.js:6:28852
c@https://cdn.melvor.net/core/v018/assets/js/oneui.core.min.js:6:79495
https://cdn.melvor.net/core/v018/assets/js/oneui.core.min.js:6:81938

### Reproduction Steps

I can go on my save on chrome but it won't let me on my phone anymore, there's an update for the app but I can't download it because of my ios version.

### Which platforms are you experiencing this bug on?

- [ ] Chrome
- [ ] Firefox
- [ ] Edge
- [ ] Safari
- [X] Mobile App (iOS)
- [ ] Mobile App (Android)
- [ ] Steam
- [ ] Other (Please Specify)

### Which version of the game are you experiencing this bug on?

Beta v1.0

### Game Subversion

Idk doesn't let me check on my phone

### Are you using any scripts or extensions?

No

**Comment by MelvorIdle** [comment]
Are you attempting to load a save that is relatively old?

**Comment by Bluebloodhero** [comment]
No, I had tried loading the game on my phone for a few days before but it kept loading. When it finally let me in, I tried to load the save and it broke

**Comment by MelvorIdle** [comment]
Do you have a cloud account? I can take a look and provide a fixed save if you do

**Comment by Bluebloodhero** [comment]
Yes I do, my username is Bluebloodhero

**Comment by MelvorIdle** [comment]
Can you please load your **cloud save** now and see if you can get into the game?

**Comment by Bluebloodhero** [comment]
I tried to load the cloud save on my phone and got a slightly different error code, would you want that?

**Comment by Bluebloodhero** [comment]
populateMasteryObject@https://ios.melvoridle.com/assets/js/game/mastery2.js?1297:4:31
updateWindow@https://ios.melvoridle.com/assets/js/game/main.js?1297:4:264
finaliseLoad@https://ios.melvoridle.com/assets/js/game/main.js?1297:372:13
selectCharacter@https://ios.melvoridle.com/assets/js/game/main.js?1297:364:192
onloadEvent@https://ios.melvoridle.com/assets/js/game/save.js?1297:61:232
https://ios.melvoridle.com/assets/js/game/save.js?1297:66:382
https://cdn.melvor.net/core/v018/assets/js/oneui.core.min.js:6:84826
each@https://cdn.melvor.net/core/v018/assets/js/oneui.core.min.js:6:2971
https://cdn.melvor.net/core/v018/assets/js/oneui.core.min.js:6:84807
r@https://cdn.melvor.net/core/v018/assets/js/oneui.core.min.js:6:28088
fireWith@https://cdn.melvor.net/core/v018/assets/js/oneui.core.min.js:6:28852
c@https://cdn.melvor.net/core/v018/assets/js/oneui.core.min.js:6:79495
https://cdn.melvor.net/core/v018/assets/js/oneui.core.min.js:6:81938
